### PR TITLE
Remove obsolete P4 settings

### PR DESCRIPTION
**Pull request by comzeradd**
- Remove Campaign styles import setting: As we phased out Campaign themes, this is no longer needed.
- Remove AB antiflicer testing: As Google Optimize is no longer a thing, we can remove this to avoid confusing. If there is another global AB testing tool in the future, it most likely require a different type implementation.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [oberon](https://www-dev.greenpeace.org/test-oberon) | [admin](https://www-dev.greenpeace.org/test-oberon/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-oberon/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-oberon) | [composer-local.json](https://github.com/greenpeace/planet4-test-oberon/blob/main/composer-local.json)

:watch: 2024.11.25 09:22:42

File: src/Settings.php [hunk 1]
```diff
@@ -425,15 +411,6 @@ public function __construct()
                         'id' => 'google_tag_manager_domain',
                         'type' => 'text',
                     ],
-                    [
-                        'name' => __('AB Hide Selector', 'planet4-master-theme-backend'),
-                        'desc' => __(
-                            'When running an AB test it is possible that the original is shown for a short while (called flicker). If this happens you can enter a CSS selector here and matching elements will be hidden until the test is fully loaded.',
-                            'planet4-master-theme-backend'
-                        ),
-                        'id' => 'ab_hide_selector',
```
Comment (mleray) [inline src/Settings.php:434]: This one should also be removed from the context, we set it [here](https://github.com/greenpeace/planet4-master-theme/blob/main/src/MasterSite.php#L700)

Comment (comzeradd) [reply src/Settings.php:434]: Ah I missed that. It's now removed.

**Review by mleray — APPROVED** [review]
Looks good to me 👍 🧹

**Pull request merged**